pkg/microservice/aslan/core/multicluster/service/clusters.go

```diff
@@ -296,6 +382,17 @@ func CreateCluster(args *K8SCluster, logger *zap.SugaredLogger) (*commonmodels.K
 	return s.CreateCluster(cluster, args.ID, logger)
 }
 
```

Comment:
1. Modify the method name to more accurately describe the content of the method.
2. Why use array instead of map / sets.string？